java-common-protos/.OwlBot-hermetic.yaml

```diff
@@ -22,50 +22,50 @@ deep-preserve-regex:
 - "/java-common-protos/google-.*/src/test/java/com/google/cloud/.*/v.*/it/IT.*Test.java"
 
 deep-copy-regex:
-- source: "/google/api/google-api-java/proto-google-api-java/src"
+- source: "/google/api/.*-java/proto-google-.*/src"
```

Comment:
@diegomarquezp could you explain why do we need to build owlbot cli source folder?

Reply:
The folder name is coming BUILD target and we need to parse BUILD.bazel to know the folder name.